virtualizarr/manifests/store.py

```diff
@@ -353,6 +342,35 @@ async def list_dir(self, prefix: str) -> AsyncGenerator[str, None]:
         for k in self._group.arrays.keys():
             yield k
 
+    @classmethod
+    def from_kerchunk_refs(
+        cls,
+        refs: KerchunkStoreRefs,
+        group: str | None = None,
+        fs_root: str | None = None,
+    ) -> ManifestStore:
+        """
+        Construct a ManifestStore from a dictionary of kerchunk references.
+
+        Parameters
+        ----------
+        refs: dict
+            The Kerchunk references, as a dictionary.
+        group: string, optional
+        fs_root: string, optional
+            The root of the fsspec filesystem on which these references were generated.
+            Required if any paths are relative in order to turn them into absolute paths (which virtualizarr requires).
+        """
+        # TODO teach this method to understand kerchunk json/parquet filepaths too?
+
+        from virtualizarr.translators.kerchunk import manifeststore_from_kerchunk_refs
+
+        return manifeststore_from_kerchunk_refs(
+            refs,
+            group=group,
+            fs_root=fs_root,
+        )
```

Comment:
Why put this as a class method on ManifestStore? It seems to me that having `manifeststore_from_kerchunk_refs` be a `create_manifest_store` function in the `kerchunk` module would be more in-line with the design proposed in https://github.com/zarr-developers/VirtualiZarr/issues/498.

Reply:
Fair point - I guess arguably `manifeststore_from_kerchunk_refs` is a reader...